shared/js/baseOptions.js

```diff
@@ -0,0 +1,149 @@
+startTiming('Puzzle', script_path, '');
```

Comment:
For that matter maybe some of these things (startTiming?) are shared between blockly and non-blockly apps

> On Jan 12, 2015, at 22:58, Josh Lory <notifications@github.com> wrote:
> 
> In shared/js/baseOptions.js:
> 
> > @@ -0,0 +1,149 @@
> > +startTiming('Puzzle', script_path, '');
> Good call, renamed to initBlockly.js.
> 
> —
> Reply to this email directly or view it on GitHub.
>